gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.4.1-bin.zip
+zipStoreBase=GRADLE_USER_HOME
```

Comment:
I'm not familiar with the zipStore settings. I have a feeling, though, that they should be left to their default value so that they are stored with the project, not globally.

Reply:
the truth is that all these lines were left at the default. I just ran the gradle wrapper command to add gradle to the project, so unless there's something that you think should specifically be different I think that all the values here should be left as is with the defaults.

Reply:
Hmmm. OK. Looks like something has changes relatively recently.